open_spiel/games/2048.cc

```diff
@@ -0,0 +1,383 @@
+// Copyright 2022 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Can you add a comment block here that describes the game?

Also, it should include (i) a link to the Wikipedia description, and (ii) a link to the code this was based on (https://github.com/gabrielecirulli/2048) depending on how much you used.

Reply:
Have added this in 2048.h, should I move it here?